corehq/apps/accounting/README.md

```diff
@@ -0,0 +1,21 @@
+### Adding a new Privilege
+
```

Comment:
With our Django upgrade almost ready, this no longer might be True at all. Please feel free to update this, if any of this is wrong.